build-config/resources/Ballerina.toml

```
@@ -1,8 +1,8 @@
 [package]
 distribution = "2201.10.0"
 org = "ballerinax"
-name = "hubspot.crm.object.tickets"
-version = "@toml.version@"
+name = "hubspot.crm.obj.tickets"
+version = "1.0.0"
```

Don't change the version in this file. If you need to change the version number, change it in the `gradle.properties` file.

```suggestion
version = "@toml.version@"
```